### PR TITLE
#27 - ConfigFile for permissions slices

### PR DESCRIPTION
**Pull request by genryxy**
Part of #27 
Create `ConfigFile` and its fake implementation. Used this interface in permissions slices.

### CODE REVIEW

**Comment by olegmoz** [comment]
@olenagerasimova could you please review this one?

File: src/main/java/com/artipie/management/api/artifactory/GetPermissionSlice.java [hunk 1]
```diff
@@ -56,23 +56,23 @@
 public final class GetPermissionSlice implements Slice {
 
     /**
-     * Artipie settings.
+     * Repository permissions.
      */
-    private final Storage storage;
+    private final RepoPermissions permissions;
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/management/api/artifactory/GetPermissionSlice.java:61]: @genryxy please, replace ; with . at the end of the sentence

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy just one cosmetic comment and a question: main implementation of the interface stays in artipie/artipie, right?

**Comment by genryxy** [comment]
@olenagerasimova thanks, yes, main implementation of the interface `ConfigFile` stays in artipie/artipie

File: src/main/java/com/artipie/management/api/artifactory/GetPermissionSlice.java (hunk 1 above)
Comment (genryxy) [reply src/main/java/com/artipie/management/api/artifactory/GetPermissionSlice.java:61]: @olenagerasimova this line is not comment. Probably the highlighted text is a bit of misleading in this case.

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good to me

**Comment by olenagerasimova** [comment]
@olegmoz please, check/merge

**Pull request merged**